src/pages/search/advanced_search/LanguageFilter.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Chip, FormControl, InputLabel, MenuItem, Select } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { ResultParam } from '../../../api/searchApi';
+import i18n from '../../../translations/i18n';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { languageOptions } from '../../registration/DescriptionPanel';
+
+export const LanguageFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const languageParam = searchParams.get(ResultParam.PublicationLanguageShould)?.split(',') || [];
+
+  const handleChange = (selectedUris: string[] | null) => {
+    if (selectedUris && selectedUris.length > 0) {
+      const languages = selectedUris
+        .map((iso6393Code) => {
+          const selectedLanguage = languageOptions.find((language) => language.iso6393Code === iso6393Code);
+          return selectedLanguage ? selectedLanguage.iso6393Code : null;
+        })
+        .filter(Boolean);
+
+      if (languages.length > 0) {
+        searchParams.set(ResultParam.PublicationLanguageShould, languages.join(','));
+      } else {
+        searchParams.delete(ResultParam.PublicationLanguageShould);
+      }
+    } else {
+      searchParams.delete(ResultParam.PublicationLanguageShould);
+    }
+
+    history.push({ search: searchParams.toString() });
+  };
+
+  const handleDelete = (iso6393CodeToDelete?: string) => () => {
+    selectedLanguages.length > 1
+      ? handleChange(languageParam.filter((iso6393Code) => iso6393Code !== iso6393CodeToDelete))
+      : handleChange([]);
+  };
+
+  const selectedLanguages = languageParam
+    .map((iso6393Code) => languageOptions.find((language) => language.iso6393Code === iso6393Code))
+    .filter(Boolean);
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', maxWidth: '15rem' }}>
+      <FormControl fullWidth sx={{ width: '15rem' }}>
+        <InputLabel>{t('registration.description.primary_language')}</InputLabel>
+        <Select
+          multiple
+          value={languageParam}
+          data-testid={dataTestId.startPage.advancedSearch.publicationLanguageField}
+          label={t('registration.description.primary_language')}
+          onChange={(event) => {
```

Comment:
føles unaturlig at den ikke lukkes når man velger ett språk IMO 🤔

Reply:
Ja, enig i det. Men der virker som om MUI har gjort dette "by design", da det ikke er noen `blur` props på komponenten. La inn en fiks med `useState`

Reply:
Sant det, mulig det faktisk er det "retteste" å la den være åpen når det er multiple faktisk? https://mui.com/material-ui/react-select/#default
Ser ikke for meg at det er mange tilfeller hvor man vil velge flere språk da for så vidt og da blir det ev. et litt kunstig ekstra trykk, så opp til deg om vi skal lukke eller ikke :)

Reply:
Etter å ha testa oppdatert branch heller jeg nok fortsatt mot at det er best å lukke i vårt tilfelle ja